source/draw/Mesh.ooc

```diff
@@ -6,9 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-version(!gpuOff) {
```

Comment:
The version flag was preventing `GpuMesh` from being used in `DrawState`. A mesh could in theory exist for CPU rendering too even if we don't have it now.